src/commands/cmd_key.cc

```diff
@@ -336,25 +336,25 @@ class CommandPersist : public Commander {
   }
 };
 
-class CommandDel : public Commander {
```

Comment:
Some problems:
- `CommandDel` cannot be removed;
- `CommandDelPrefix` should be in this file, instead of another file (if you really build the project with your changes, I think you can find compile errors);
- There should be some test case for this new command